api/movieApi.ts

```diff
@@ -0,0 +1,29 @@
+import api from './index';
+
+const API_KEY = '6d482b614139b65c7359f30521c4deb2';
+
+export const getNowPlaying = () => {
+  return api
+    .get(`movie/now_playing?api_key=${API_KEY}`)
+    .then((res) => res.data);
+};
+
+export const getTopRated = () => {
+  return api.get(`movie/top_rated?api_key=${API_KEY}`).then((res) => res.data);
+};
+
+export const getPopular = () => {
+  return api.get(`movie/popular?api_key=${API_KEY}`).then((res) => res.data);
+};
+
+export const getUpcoming = () => {
+  return api.get(`movie/upcoming?api_key=${API_KEY}`).then((res) => res.data);
+};
+
+export const getTopTv = () => {
+  return api.get(`tv/popular?language=en-US&page=1`).then((res) => res.data);
+};
```

Comment:
요 4개의 함수가 사실상 같은 기능이여서 하나의 함수로 만든 후, 외부 컴포넌트에서 url 을 넣어 데이터를 가져오는 식의 로직도 괜찮을 것 같아요!!

```suggestion

export const getMovies(url:string) => {
    return api.get(url+`?api_key=${API_KEY}`).then((res) => res.data);
}

......

<Component data={await getMovies('movie/top_rated')} />